package.json

```diff
@@ -32,7 +32,7 @@
     "web3": "^1.3.4"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "env-cmd -f .env.dev react-scripts start",
```

Comment:
can't this work w craco?

Reply:
if .env.dev is mandatory to run the project, needs to be documented in the readme. Running yarn start now fails